### PR TITLE
*: unify the use of PD API URL constants

### PR DESCRIPTION
**Pull request by JmPotato**
### What problem does this PR solve?

Issue Number: ref #35319.

### What is changed and how it works?

Unify the use of PD API URL constants to make sure it can be maintained more easily.

### Check List

Tests 

- [x] Unit test
- [x] Integration test

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by hawkingrei** [comment]
Great job. 

It is a better way to supply a pd client library for tidb.

**Review by hawkingrei — APPROVED** [review]

**Comment by CabinfeverB** [comment]
> Great job.
> 
> It is a better way to supply a pd client library for tidb.

a HTTP client?

**Comment by hawkingrei** [comment]
> > Great job.
> > It is a better way to supply a pd client library for tidb.
> 
> a HTTP client?

Yes, but it is enough for this pr.

File: pkg/util/pdapi/const.go [hunk 1]
```diff
@@ -14,17 +14,86 @@
...
+	HotRead                = "/pd/api/v1/hotspot/regions/read"
+	HotWrite               = "/pd/api/v1/hotspot/regions/write"
+	HotHistory             = "/pd/api/v1/hotspot/regions/history"
+	Regions                = "/pd/api/v1/regions"
+	StoreRegions           = "/pd/api/v1/regions/store"
+	ScanRegions            = "/pd/api/v1/regions/key"
+	EmptyRegions           = "/pd/api/v1/regions/check/empty-region"
+	AccelerateSchedule     = "/pd/api/v1/regions/accelerate-schedule"
+	RegionByID             = "/pd/api/v1/region/id"
+	RegionKey              = "/pd/api/v1/region/key"
```
Comment (nolouch) [inline pkg/util/pdapi/const.go:33]:
```suggestion
	RegionByKey              = "/pd/api/v1/region/key"
```

File: pkg/util/pdapi/const.go [hunk 2]
```diff
@@ -14,17 +14,86 @@
...
+	PProfGoroutine         = "/pd/api/v1/debug/pprof/goroutine"
 )
+
+// ConfigWithTTLSeconds returns the config API with the TTL seconds parameter.
+func ConfigWithTTLSeconds(ttlSeconds float64) string {
+	return fmt.Sprintf("%s?ttlSecond=%.0f", Config, ttlSeconds)
+}
+
+// StoreWithID returns the store API with store ID parameter.
+func StoreWithID(id uint64) string {
```
Comment (CabinfeverB) [inline pkg/util/pdapi/const.go:67]: How about `StoreByID` which is same with `RegionByID`

**Review by CabinfeverB — APPROVED** [review]

**Comment by JmPotato** [comment]
@lance6716 @Leavrth Please help take a look, thx.

**Review by lance6716 — APPROVED** [review]

**Review by Leavrth — APPROVED** [review]

**Pull request merged**